packages/worker/src/utilities/email.ts

```diff
@@ -106,8 +91,8 @@ async function buildEmail(
     throw "Unable to build email, missing base components"
   }
 
-  let name = user ? user.name : undefined
-  if (user && !name && user.firstName) {
+  let name: string | undefined
```

Comment:
this change, just to check, `user.name` - is that a property that was ever populated/even exists? I'm guessing that is why it was removed from here as I don't think thats a prop our users have

Reply:
As far as I can tell it doesn't exist.